### PR TITLE
Port alpine and doc 9863 to 4.0

### PR DESCRIPTION
**Pull request by osfameron**
Port #89 and DOC-9863 from 3.2

### CODE REVIEW

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 1]
```diff
@@ -16,19 +16,64 @@ The Couchbase PHP SDK is compatible with PHP 7.3+.
 
 == Installing the SDK
 
-Before installing the PHP SDK, you must install the xref:3.2@c-sdk:hello-world:start-using-sdk.adoc[C SDK -- _libcouchbase_ (LCB)];
+The Couchbase PHP SDK depends on the xref:3.2@c-sdk:hello-world:start-using-sdk.adoc[C SDK -- _libcouchbase_ (LCB)];
```
Comment (RichardSmedley) [inline modules/hello-world/pages/start-using-sdk.adoc:19]: I'll let this through, as I'll be updating later  :+1:

Comment (osfameron) [reply modules/hello-world/pages/start-using-sdk.adoc:19]: indeed! 👍

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -17,14 +17,12 @@ For release notes, download links, and installation methods for 2.6 and earlier
...
-=== Installing on Linux-like systems
-
-// needs updating for 3.0
+=== Installing on Mac and Linux systems
 
 Before installing the PHP SDK, you must install the xref:3.2@c-sdk:hello-world:start-using-sdk.adoc[C SDK -- _libcouchbase_ (LCB)];
 version 3.2.0 or higher is required for PHP Client 3.2.
 
-After installing the C SDK, install the PHP SDK through your PHP distribution's pecl command:
+After installing `libcouchbase`, install the Couchbase PHP SDK through your PHP distribution's `pecl` command:
```
Comment (RichardSmedley) [inline modules/project-docs/pages/sdk-release-notes.adoc:25]: I'll also let this through here, as I'll be updating this one later too :+1: +1

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Pull request merged**